crates/iota-core/src/congestion_tracker.rs

```diff
@@ -12,22 +12,19 @@ use iota_types::{
     transaction::{TransactionData, TransactionDataAPI},
 };
 use moka::{ops::compute::Op, sync::Cache};
+use ordered_float::OrderedFloat;
 
 use crate::execution_cache::TransactionCacheRead;
 
 /// Capacity of the congestion tracker's cache.
 const CONGESTION_TRACKER_CACHE_CAPACITY: u64 = 10_000;
 
-/// Alias type for holding transaction's gas price and mutable (or
-/// congested) shared objects.
-type TransactionGasPriceMutSharedObjectsPair = (u64, Vec<ObjectID>);
-
 /// Alias for type holding congestion info per checkpoint.
 type CongestionInfoMap = HashMap<ObjectID, CongestionInfo>;
 
 /// Holds tracked per-object congestion info.
 #[derive(Clone, Copy, Debug)]
-struct CongestionInfo {
+pub struct CongestionInfo {
```

Comment:
```suggestion
struct CongestionInfo {
```
Used only in this module, so does not need to be public.